packages/core/src/asciidoctor-core-api.js

```diff
@@ -1857,6 +1857,41 @@ Document.prototype.hasHeader = function () {
   return this['$header?']()
 }
 
+/**
+ * Replay attribute assignments at the block level
+ *
+ * @param {Object} blockAttributes - A JSON of attributes
+ *
+ * @memberof Document
+ */
+Document.prototype.playbackAttributes = function (blockAttributes) {
+  blockAttributes = toHash(blockAttributes)
+  if (blockAttributes) {
+    var attrEntries = blockAttributes['$[]']('attribute_entries')
+    if (attrEntries && Array.isArray(attrEntries)) {
+      var result = []
+      for (var i = 0; i < attrEntries.length; i++) {
+        var attrEntry = attrEntries[i]
+        var attrEntryObject = attrEntry
```

Comment:
Why two variables?  (also why not const?)

Reply:
It's a mistake, nice catch 👍